src/Mod/Path/PathScripts/PathAdaptiveGui.py

```diff
@@ -214,6 +214,10 @@ def getFields(self, obj):
         obj.setEditorMode('Stopped', 2)  # hide this property
 
 
-Command = PathOpGui.SetupOperation('Adaptive', PathAdaptive.Create, TaskPanelOpPage,
-                                   'Path-Adaptive', QtCore.QT_TRANSLATE_NOOP("PathAdaptive", "Adaptive"),
-                                   QtCore.QT_TRANSLATE_NOOP("PathPocket", "Adaptive clearing and profiling"))
+Command = PathOpGui.SetupOperation('Adaptive',
+        PathAdaptive.Create,
+        TaskPanelOpPage,
+        'Path_Adaptive',
+        QtCore.QT_TRANSLATE_NOOP("Path_Adaptive", "Adaptive"),
+        QtCore.QT_TRANSLATE_NOOP("Path_Pocket", "Adaptive clearing and profiling")
```

Comment:
The translation context is `Path_Pocket` but it looks like it should be `Path_Adaptive`. If so, this was a pre-existing mistake in the code. Shall I fix?

Reply:
I think you are correct (comparing, e.g. to PathCustomGui.py lines 80 and 81), but changing this will require changing all of the .ts files, too, as far as I know. Then again, the UK .ts entry for the adaptive clearing and profiling looks very suspect!

Reply:
There is another pending PR (#4093 ) that will change all the .ts files, so no worries. This should be fixed since it helps the translators understand the context of the translation. I'll make the change now.

Reply:
Fixed in 26a434b